src/main/resources/templates/lsp/stylelint-lsp/installer.json

```diff
@@ -0,0 +1,36 @@
+{
+  "id": "stylelint-lsp",
+  "name": "StyleLint Language Server",
+  "executeOnStartServer": false,
+  "properties": {
+    "workingDir" : "$USER_HOME$/.lsp4ij/lsp/stylelint-lsp/node_modules"
+  },
+  "check": {
+    "exec": {
+      "name": "Trying current command",
+      "command": "${server.command}",
+      "timeout": 2000
+    }
+  },
+  "run": {
+    "exec": {
+      "name": "Install StyleLint Language Server",
+      "workingDir": "${workingDir}",
+      "ignoreStderr": true,
+      "command": {
+        "windows": "npm.cmd install stylelint-lsp --force",
+        "default": "npm install stylelint-lsp --force"
+      },
+      "onSuccess": {
+        "configureServer": {
+          "name": "Configure StyleLint Language Server command",
+          "command": {
+            "windows": "npx.cmd stylelint-lsp --stdio",
```

Comment:
Could we not use a js file like typescript https://github.com/redhat-developer/lsp4ij/blob/22b88d1beeafad5a618e09a1888ac53ad2f6061e/src/main/resources/templates/lsp/typescript-language-server/installer.json#L28 

In your case you are using stylelint-lsp registered in global registry and nor form the workingdir